.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text eol=lf
+*.jpg binary
+*.png binary
```

Comment:
Just a hint, use a global `.gitattributes` https://stackoverflow.com/questions/28026767/where-should-i-place-my-global-gitattributes-file

Reply:
Thanks

Reply:
after reading your link, i think i placed it right. 
because in other projects i need other line endings and 
in this project we have a check on LF line endings